packages/libs/coreui/src/components/containers/NoteBox.tsx

```diff
@@ -0,0 +1,47 @@
+import { css } from '@emotion/react';
+import React, { ReactNode } from 'react';
+import { error, warning, mutedBlue } from '../../definitions/colors';
+
+export interface Props {
+  type: 'info' | 'warning' | 'error';
+  children: ReactNode;
+}
+
+const baseCss = css`
```

Comment:
Perhaps additionally a default font? I believe coreui generally uses Inter or Roboto

Reply:
For content, I think we should use the same font as the rest of the site, but I will take a look at using one of these other fonts.

Reply:
I think I'm going to leave the font as-is.